cmd/k3s/main.go

```diff
@@ -35,10 +35,10 @@ func main() {
 		cmds.NewCRICTL(externalCLIAction("crictl", dataDir)),
 		cmds.NewCtrCommand(externalCLIAction("ctr", dataDir)),
 		cmds.NewCheckConfigCommand(externalCLIAction("check-config", dataDir)),
+		cmds.NewEtcdSnapshotCommand(wrap("k3s-"+cmds.EtcdSnapshotCommand, dataDir, os.Args)),
```

Comment:
It's ugly, but server and agent don't have a const for the command string so you could drop it if you want, or do it like this:
```suggestion
		cmds.NewEtcdSnapshotCommand(wrap(version.Program+"-"+cmds.EtcdSnapshotCommand, dataDir, os.Args)),
```